frappe/utils/install.py

```diff
@@ -330,3 +335,23 @@ def add_standard_navbar_items():
 		navbar_settings.append("help_dropdown", item)
 
 	navbar_settings.save()
+
+def get_custom_navbar_items(navbar_items, standard_items):
+	# hack to maintain the order of custom and standard items
+	counter = 0
+	for d in navbar_items:
+		if not d.is_standard:
+			row = {
+				"item_label": d.item_label,
+				"item_type": d.item_type,
+				"route": d.route,
+				"action": d.action,
+				"is_standard": 0,
+				"idx": d.idx,
+			}
+			standard_items.insert(d.idx - 1, row)
```

Comment:
This insert can cause index error. 


Also this isn't reliable too. 

I tried yet-another-approach in https://github.com/frappe/frappe/pull/16827 by adding unique identifier to navbar items. That's also problematic because other apps have added standard navbar items (like ERPNext) -_-